mbuild/coordinate_transform.py

```diff
@@ -16,7 +16,9 @@
 ]
 
 
-def force_overlap(move_this, from_positions, to_positions, add_bond=True):
+def force_overlap(
+    move_this, from_positions, to_positions, add_bond=True, reset_labels=False
```

Comment:
Does that mean you agree to keep it False? I did this to avoid the "moving goal post" that my issue described.

Reply:
I was thinking it should stay True. While I agree that it is a bit of a strange behavior to have the labels reset every time, I do think the reset happens in a consistent way each time, and that potentially other recipes relied upon this feature, since it was so hard coded into the force_overlap or remove function. I'd be worried that changing the default could lead to unexpected bugs for people who aren't aware of the change. I'm open to either way though, since I do agree starting from scratch I'd set it to False as the default. What do you think @chrisjonesBSU?

Reply:
I agree with Cal. Thanks to this PR, it's easy enough to change the behavior to not reset the labels, we might as well keep the original behavior to avoid breaking changes. Maybe we can add a Notes section to `force_overlap` that quickly explains the use case of setting `reset_labels=False`